docs/config.md

```diff
@@ -297,6 +297,23 @@ export default {
 };
 ```
 
+## duckdb <a href="https://github.com/observablehq/framework/pull/1734" class="observablehq-version-badge" data-version="prerelease" title="Added in #1734"></a>
+
+The **duckdb** option specifies the list of DuckDB [extensions](./sql#extensions) that you want to self-host and make available in the `sql` and `DuckDBClient` instances.
+
+Its **install** key is an array of the names of extensions to self-host; it defaults to `["json", "parquet"]`. The optional **load** key is an array of names of extensions to load immediately. It defaults to the empty array (since "json" and "parquet" are autoloaded, there is no reason to load them before we actually need them). Lastly, the **source** key is an object of key:value pairs representing the repo used to download each extension. The source repo for any name defaults to `core`, which points to `https://extensions.duckdb.org/`. You can use `core`, `community` (which points to `https://community-extensions.duckdb.org/`), or a custom URL:
+
+```js run=false
+duckdb: {
+  install: ["json", "spatial", "h3", "custom"],
+  load: ["spatial"],
+  source: {
+    h3: "community",
+    custom: "https://my-custom-repo.tld"
+  }
+}
```

Comment:
Have you had a chance to consider this feedback yet?

Reply:
I've now added tests and updated the documentation.

In addition, I have decoupled install and load. You can now *load* an extension from a remote source, for example if you want to create a "spatial"+"h3" sql template literal without self-hosting h3. (This might be more useful for users who develop their own extensions and want to serve them from a custom extensions repo, rather than from a local self-hosted copy.)